Dockerfile

```diff
@@ -1,10 +1,13 @@
 FROM ghcr.io/coqui-ai/stt-train:main
 
 RUN python -m pip install pandarallel
+RUN python -m pip install stt
+RUN python -m pip install numpy
+RUN python -m pip install librosa
 
 WORKDIR /home/ubuntu
 COPY . /home/ubuntu/
 
```

Comment:
```suggestion
RUN mkdir /home/ubuntu/stt_model
RUN wget https://github.com/coqui-ai/STT-models/releases/download/english%2Fcoqui%2Fv1.0.0-huge-vocab/model.tflite -P /home/ubuntu/stt_model
RUN wget https://github.com/coqui-ai/STT-models/releases/download/english%2Fcoqui%2Fv1.0.0-huge-vocab/huge-vocabulary.scorer -P /home/ubuntu/stt_model
```

Reply:
@loganhart420 let's grab the acoustic and language model via the dockerfile

Reply:
Thanks I wasn't sure how to handle it